src/main/java/com/twentyonec/ItemsLogger/commands/Reload.java

```diff
@@ -0,0 +1,19 @@
+package com.twentyonec.ItemsLogger.commands;
+
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandSender;
+
+import com.twentyonec.ItemsLogger.ItemsLogger;
+
+public class Reload implements CommandInterface{
```

Comment:
Descriptive class names are good practice. `Reload` is very vague
```suggestion
public class ReloadConfig implements CommandInterface {
```